webpack/makeConfig.js

```diff
@@ -94,6 +94,7 @@ export default function makeConfig(isDevelopment) {
         new webpack.DefinePlugin({
           'process.env': {
             NODE_ENV: JSON.stringify(isDevelopment ? 'development' : 'production'),
+            DEVTOOLS: isDevelopment,
```

Comment:
@ondrejbartas I think that if the condition is not simple `if (false)` then the code will not be removed by Uglifier, right?